tests/end_to_end/features/steps/autovariants.py

```
@@ -26,31 +26,56 @@ def step_impl(context):
-    'I should be able to reuse the same variant for the same "{transformation_name}" transformation'
+    'I should be able to reuse the same variant for the same "{transformation_name}" transformation of type "{tf_type}"'
 )
-def step_impl(context, transformation_name):
-    @context.spark.df_transformation(
-        name=transformation_name,
-        inputs=[context.transactions],
-    )
-    def some_transformation(df):
-        """Unedited transactions"""
-        return df
+def step_impl(context, transformation_name, tf_type):
```

Ruff (0.8.2)

Redefinition of unused `step_impl` from line 32

(F811)